desktop/config/index.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
```

Comment:
Are we using this elsewhere in the app? Do we even need to?

I _am_ aware that certain parts of the code can't be run through Babel and what not, so please pardon me if this _has_ to be vanilla JS.